azure-proto-network/NetworkSecurityGroupOperations.cs

```diff
@@ -1,61 +1,68 @@
-﻿using Azure;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure;
 using Azure.ResourceManager.Network;
 using Azure.ResourceManager.Network.Models;
 using Azure.ResourceManager.Core;
-using System.Collections.Generic;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace azure_proto_network
 {
     /// <summary>
-    /// An operations + Data class for NSGs
-    /// TODO: How does the operation signature change for resources that support Etags?
+    /// A class representing the operations that can be performed over a specific NetworkSecurityGroup.
     /// </summary>
-    public class NetworkSecurityGroupOperations : ResourceOperationsBase<NetworkSecurityGroup>, ITaggableResource<NetworkSecurityGroup>, IDeletableResource
+    public partial class NetworkSecurityGroupOperations : ResourceOperationsBase<NetworkSecurityGroup>, ITaggableResource<NetworkSecurityGroup>, IDeletableResource
```

Comment:
Do we need this to be a partial class?
```suggestion
    public class NetworkSecurityGroupOperations : ResourceOperationsBase<NetworkSecurityGroup>, ITaggableResource<NetworkSecurityGroup>, IDeletableResource
```